src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -94,6 +94,17 @@ public GHTeam getTeamByName(String name) throws IOException {
     }
 
     /**
+     * Finds a team that has the given slug in its {@link GHTeam#getSlug()}
+     */
+    public GHTeam getTeamBySlug(String slug) throws IOException {
+        for (GHTeam t : listTeams()) {
+            if(t.getSlug().equals(slug))
```

Comment:
I just opened a feature request with GitHub support to allow selecting teams via `:slug` key in addition to the `:id` key within the API.